subsys/usb/usb_c/Kconfig

```diff
@@ -39,6 +39,17 @@ config BUILD_OUTPUT_VIF
 	help
 	  Generate XML file containing VIF policies during project build.
 
+config GENVIF_INPUT_VIF_XML_PATH
+	string "Path to input VIF file"
+	depends on BUILD_OUTPUT_VIF
+	help
+	  Pass custom path of input VIF XML file residing either in
+	  `APPLICATION_SOURCE_DIR` or `BOARD_DIR` to GenVIF script. Consider
+	  `APPLICATION_SOURCE_DIR` and `BOARD_DIR` as source directory. If this config
```

Comment:
Is this meant to say that this config is interpreted as a subdiractory of the application path or board path? I don't think it's very clear right now. Also I would avoid referring to `APPLICATION_SOURCE_DIR` or `BOARD_DIR` as that makes it looks like they are `CONFIG_` options, which they are not.

Reply:
Yes, the application directory is considered a source directory for Kconfig option, I removed the variables from description in new revision.